ENGAGEHF/Bluetooth/Views/MeasurementRecordedView.swift

```diff
@@ -0,0 +1,123 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziViews
+import SwiftUI
+
+
+struct MeasurementHeader: View {
+    @Environment(\.dismiss) var dismiss
+    @Binding var viewState: ViewState
+    
+    
+    var body: some View {
+        ZStack {
+            HStack {
+                Button(action: {
+                    dismiss()
+                }) {
+                    Text("DISMISS_BUTTON")
+                        .foregroundStyle(Color.accentColor)
+                }
+                    .buttonStyle(PlainButtonStyle())
+                    .padding(.trailing, 10)
+                    .disabled(viewState != .idle)
+                    .accessibilityLabel("Dismiss")
+                
```

Comment:
I would suggest removing empty lines within a view body or only rarely use them to highlight larger parts of the views to be separate. If that's the case it might be an indication that the view should be split in multiple smaller views.